cli/.js/src/main/scala/aqua/FuncCompiler.scala

```diff
@@ -35,7 +35,9 @@ class FuncCompiler[F[_]: Files: AquaIO: Async](
       .collectFirstSome(_.allFuncs.get(func.name))
       .map(validNec)
       .getOrElse(
-        Validated.invalidNec[String, FuncArrow](s"There is no function called '${func.name}'")
+        Validated.invalidNec[String, FuncArrow](
+          s"There is no function called '${func.name}'. Maybe this function is not exported and you need to add 'export ${func.name}?'"
```

Comment:
It's not very helpful, and it's also contradictory. 

1. "There is no function, maybe it's not exported?" - this message contradicts itself. Either there is no function, or there is but not exported. Is it possible to check if such function exists, but not exported? 
2. "You need to add `export func.name`" not clear where to add that. To function definition?

If you can't do `1.`, then I'd suggest to give the following error message
`"There is no exported function '${func.name}'. See fluence.dev/aqua-book/... on exporting functions"